breadbox/breadbox/compute/analysis_tasks.py

```diff
@@ -337,16 +336,7 @@ def run_custom_analysis(
             analysis_type == models.AnalysisType.pearson
             or analysis_type == models.AnalysisType.association
         ):
-            if query_node_id:
-                # This is the node for the feature of the vector we're querying
-                dataset_feature_id = query_node_id
-                feature_dataset = dataset_crud.get_dataset(db, user, dataset_feature_id)
-                if feature_dataset is None:
-                    raise ResourceNotFoundError("Query dataset not found")
-                query_series = get_feature_data_slice_values(
-                    db, user, dataset_feature_id, feature_dataset, filestore_location
-                )
-            elif query_feature_id and query_dataset_id:
+            if query_feature_id and query_dataset_id:
                 # The given query Id should be the id of the feature itself
```

Comment:
I realize you didn't write this recently, but when this comment says "The given query id", do you know which id it's referring to? query_feature_id?